cmd/osquery-perf/agent.go

```diff
@@ -1603,21 +1625,21 @@ func (a *agent) DistributedWrite(queries map[string]string) error {
 	return nil
 }
 
-func (a *agent) scheduledQueryResults(packName, queryName string, numResults int) json.RawMessage {
+func scheduledQueryResults(packName, queryName string, numResults int) json.RawMessage {
 	return json.RawMessage(`{
-  "snapshot": [` + rows(numResults, a.UUID) + `
+  "snapshot": [` + rows(numResults) + `
   ],
   "action": "snapshot",
   "name": "pack/` + packName + `/` + queryName + `",
-  "hostIdentifier": "` + a.UUID + `",
+  "hostIdentifier": "EF9595F0-CE81-493A-9B06-D8A9D2CCB952",
```

Comment:
Everything will come from the same host now?

Reply:
We are not processing these fields on Fleet, so to simplify am hardcoding them.